docs/contribute/app-handlers.mdx

```diff
@@ -89,18 +89,6 @@ Here is a step-by-step guide:
         self._tables = {}
     ```
 
-* Implementing the [`_register_table()`](https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/libs/api_handler.py#L164) method:
```

Comment:
So this `_register_table()` function exists in the code, but it is not used anymore.

Where can I find `the list of functions that need to be implemented for an App handler` that you mention?

Reply:
That's true. Ok, if it doesn't need to be overwritten by the child classes, then let's remove it.